sandbox/modules/identity/src/main/java/org/opensearch/identity/SecurityRestFilter.java

```diff
@@ -80,19 +80,16 @@ private boolean checkAndAuthenticateRequest(RestRequest request, RestChannel cha
             jwtClaims.put("sub", "subject");
             jwtClaims.put("iat", Instant.now().toString());
             String encodedJwt = JwtVendor.createJwt(jwtClaims);
-            String requestInfo = String.format(
```

Comment:
Did you get the latest changes from `feature/identity`? This looks like it's from an older commit.

Reply:
I think so, but if this seems like it is sneaking in somehow I will revert it. I will rebase just in case

Reply:
Looks like this change has not been addressed yet. This should resemble  `feature/identity` branch: https://github.com/opensearch-project/OpenSearch/blob/feature/identity/sandbox/modules/identity/src/main/java/org/opensearch/identity/SecurityRestFilter.java

Reply:
This should be changed back. It can be addressed in a future PR.

Reply:
Sorry, so you want me to re-open the PR to revert this--or is there a way to change it back without doing that?

Reply:
@scrawfor99 It can be addressed in a future PR. This should remain as a debug logging statement.

Reply:
Alright sounds good.